spring-boot-project/spring-boot-actuator/src/test/java/org/springframework/boot/actuate/metrics/export/prometheus/PrometheusScrapeEndpointIntegrationTests.java

```diff
@@ -108,12 +108,12 @@ void scrapePrefersToProduceOpenMetrics100(WebTestClient client) {
 	@WebEndpointTest
 	void scrapeWithIncludedNames(WebTestClient client) {
 		client.get()
-			.uri("/actuator/prometheus?includedNames=counter1_total,counter2_total")
+			.uri("/actuator/prometheus?includedNames=counter1,counter2")
```

Comment:
The implementations of scrape endpoints provided by the Prometheus client support a different query parameter format for filtering by name on the scrape endpoint as now documented at https://prometheus.github.io/client_java/exporters/filter/.
I don't know if that necessarily means Boot's endpoint should align with that, but it may be something to consider.